### PR TITLE
gh-85455: Add missing doc strings and improve docs

### PR DESCRIPTION
**Pull request by nanjekyejoannah**
Do some more clean up of `imghdr.py`.

https://bugs.python.org/issue41283

* Issue: gh-85455

### CODE REVIEW

**Review by ammaraskar — APPROVED** [review]
Looks good to me and reads clearer.

File: Doc/library/imghdr.rst
Comment (csabella) [inline Doc/library/imghdr.rst]: Perhaps use the imperative form of the verbs, that is, `Test` and `return`?  Also, the comma before the `and` isn't needed.

Comment (nanjekyejoannah) [reply Doc/library/imghdr.rst]: I find Docs use *Tests* and Doc strings uses *Test*

File: Lib/imghdr.py
Comment (facundobatista) [inline Lib/imghdr.py]: Please adapt PEP 257 in this docstring and the ones below.

In this case, just put everything in the same line and it will be ok.

File: Lib/imghdr.py
Comment (facundobatista) [inline Lib/imghdr.py]: Here and the next docstrings, please end them with a dot. But also, express what the function does.

E.g. "Verify if the image is a PNG."

**Review by facundobatista — CHANGES_REQUESTED** [review]
Thanks for this work! There I added some details regarding the docstrings, thanks!

**Comment by hugovk** [comment]
Note the imghdr module is deprecated in 3.11 and set for removal in 3.13.

See [PEP 594 – Removing dead batteries from the standard library](https://peps.python.org/pep-0594/) and https://github.com/python/cpython/issues/91217.

**Comment by nanjekyejoannah** [comment]
@iritkatriel and @hugovk   So why is it wrong to add the changes, when the module is not removed yet? IIRC we are not yet on version 3.13, no?

**Comment by iritkatriel** [comment]
It’s not wrong, it’s low priority.

Our intention is to reduce the PR backlog and sometimes that means to make a call that a low priority PR that nobody cared to merge for years is not worth investing core dev time in. Surely we won’t agree on each individual case.  Feel free to reopen and merge this if you want to.

**Comment by hugovk** [comment]
Yeah, this is a low-risk docs changes so I think it's fine to merge.

@nanjekyejoannah Would you like to address the changes suggested by Facundo?

**Comment by nanjekyejoannah** [comment]
I have tracked this as a task for myself for now, I will do the needful and merge the PR in a day or two. Thanks.

**Pull request merged**
